### ISSUE TITLE
Marble diagrams incorrectly including ) from value groups in tick counts

### ISSUE THREAD

**Issue by benlesh** [ReactiveX/rxjs#3254]
**RxJS version:** 5.x

**Code to reproduce:**

When using the test scheduler create a marble-based observable with a group

More details can be found in this PR: #2932, which we are closing in favor of @kwonoj's upcoming merging of rx-sandbox.

Basically marbles that look like this:

`---(ab)---(cd)--|`
**Expected behavior:**
```
{ time: 30, type: 'N', value: a },
{ time: 30, type: 'N',  value: b },
{ time: 70, type: 'N', value: c },
{ time: 70, type: 'N',  value: d },
{ time: 100, type: 'C' },
```
**Actual behavior:**
```
{ time: 30, type: 'N', value: a },
{ time: 30, type: 'N',  value: b },
{ time: 80, type: 'N', value: c },
{ time: 80, type: 'N',  value: d },
{ time: 110, type: 'C' },
```

**Additional information:**
The above structures are just pseudo code (clearly) lol.

See #2932 

Also use rx-sandbox for marble tests, if you even need them at all.

**Comment by benlesh** [comment]
This is apparently handled by rx-sandbox, which we're going to move into RxJS proper.

Assigning to @kwonoj 

cc @jayphelps

**Comment by kwonoj** [comment]
I'll propose small pr convert one of our tests using new module as devdeps. (eventually it could go in monorepo, or other ways around but as an immediate workaround.)

**Comment by kazinov** [comment]
Hello,

I started writing marble tests for my application and I realised that I can't test observables like this:
```
const observable$ = of(1, 2).concat(of(3, 4).delay(20));
```
If I understood correctly, the current marbles syntax just doesn't allow me to do this. 
I guess it should be ```'(ab)-(cd)'```, but it's not (c and d emitted on frame 50, not 20).

What is the recommend way to work around it? 
Is there any plan to fix this problem or should I start using rx-sandbox? 

I also created a [stack overflow question](https://stackoverflow.com/questions/50406951/how-to-test-observables-which-emit-grouped-events-with-rxjs-marbles)

Thanks!
